GOVERNANCE.md

```diff
@@ -0,0 +1,54 @@
+The discussion of the model modifications shall further create awareness of currently ongoing modeling activities across modelers and use cases to avoid duplicate models and development for the same topic.
+
+### MS2 Valid Model
+In the next step, the assigned domain experts and the modeling experts perform the modeling work and create a solution to fulfill the initial request.
+This work happens on a dedicated development branch preferably in a forked repository (for details regarding Forks in GitHub consult the [GitHub documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks)).
+As a result of this phase, the model developers propose the changes by creating a Pull-Request (PR) to commit the content of the new or updated model to the main-branch of the Tractus-X model repository.
+To create a PR, select "New pull request" in the "Pull requests" tab.
+The PR should link the initial issue requesting the change, e.g., by mentioning #{Issue-Number} in the description.
+There is a template for this PR, which contains a checklist of modeling-specific guidelines.
+It is the task of the modeling expert to check that the proposed model conforms to all items and best practices in the modeling checklist. 
+To indicate the conclusion of the modeling phase and that the model is ready for release and adoption, the modeling expert adds the label "MS2-Approved" to the PR.
+### MS3 Release Model
+The domain expert then checks with the use case team from which the initial modeling request originated to whether the proposed changes fulfill the requirements and that the use case will use it. There is a MS3 checklist in the PR template to guide the approval. This checklist, for example, asks whether there is at least one data provider and one data consumer planing to exchange data based on the definition in the proposed model. There is no specific process for getting the approval for the use case since this strongly depends on its working mode. However, one proven practice is to post the model and visualization in the working space of the use case team and ask for comments during a dedicated review period which could be one week. After the approval of the use case, the modeling team checks during one of its meetings that the proposed changes align with the other models in Tractus-X and that the model is ready for adoption from a modeling standpoint. The discussion of the model also helps the modeling team members to keep track of current modeling activities. If the use case and the modeling team approve the PR, the modeling team assigns the label "MS3-Approved" and merges the PR into the main branch of this Tractus-X models repository.
+
+## Labels
+To fascilitate the governance and keep track of ongoing PRs and issue in GitHub we use a set of labels as described in the following table: 
+
+| label | description |
+----- | -------
```

Comment:
do we need to define those labels in the eclipse repo? I assume so

Reply:
Good point, yes

Reply:
just added the labels